### ISSUE TITLE
Zoom In and Out shortcuts on macOS are not working (command + and command -)

### ISSUE THREAD

**Issue by jas-atwal** [lensapp/lens#2004]
**Describe the bug**
Zoom In and Out shortcuts on macOS are not working (command + and command -)

**To Reproduce**
1. Open Lens on Apple Macbook (I have a MacBook Pro 15" 2017 with macOS Big Sur 11.1)
2. Select 'command & +' or 'command and -) to zoom in/out

You will notice that nothing happens!

**Expected behaviour**
I would expect the UI to zoom in and out

**Workaround**
Steps to workaround the behaviour:
1. Select the 'view' menu
2. Select the 'Zoom in' or 'Zoom out' menu

**Environment (please complete the following information):**
- Lens Version: 4.0.6
- OS: macOS Big Sur 11.1
- Installation method (e.g. snap or AppImage in Linux): dmg download.

**Comment by jim-docker** [comment]
You may need to hold shift down as well

**Comment by jas-atwal** [comment]
@jim-docker Yes, that seems to have done the trick.  With all apps that I have worked with it seems that a simple command and + or - zooms in and out respectively.  However I have just come across this on the Apple website.

https://support.apple.com/en-gb/HT201236

Shift-Command-Minus sign (-): Decrease the size of the selected item.
_Shift-Command-Plus sign (+): Increase the size of the selected item._ **Command-Equals sign (=) performs the same function.**

For zooming out using the above command does not work, only 'Command and -' works.

So there are some issues/discrepancies with shortcut commands on macOS machines.

**Comment by jim-docker** [comment]
Thanks, Command-+ is a bit ambiguous since to enter '+' you must shift the key. We'll consider supporting Command-Equals, like many other apps seem to do.
